Package.swift

```diff
@@ -1,4 +1,4 @@
-// swift-tools-version:5.7
+// swift-tools-version:5.10
```

Comment:
We're technically supporting 5.7+... To be honest I think it might be good to bump the requirement here since we're able to do so actually given current users.

We might need to think about requiring 6.0 soon though, because of new language features we'd like to use, but we'll need a stable release.

Reply:
Yeah, we couldn't bump minimal macOS version without bumping swift-tools as well, it should be at least `5.9` 😔

Reply:
Sorry for the long delay here, I'm planning to get this merged now very soon. Travel and sickness got in the way, sorry for the wait